dash/App/Dapps/index.js

```diff
@@ -91,26 +91,26 @@ class _OriginModule extends React.Component {
 
 const OriginModule = Restore.connect(_OriginModule)
 
+const NetworkOrigins = ({ network, origins }) => 
+  <>
+    <div className='originTitle'>{network.name}</div>
+    {origins.map((origin) => <OriginModule origin={origin} />)}
+  </>
+
 class Dapps extends React.Component {
   render () {
+    const allOrigins = this.store('main.origins')
+    const enabledNetworks = Object.entries(this.store('main.networks.ethereum')).filter(([[networkId]]) => this.store('main.networks.ethereum', networkId, 'on'))
```

Comment:
if we don't need the keys from the networks object we can just filter on the values